privaterelay/migrations/0002_auto_20200522_1722.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.10 on 2020-05-22 17:22
```

Comment:
Would be VERY nice if we can use descriptive names like `0002_add_fxa_uid_field_in_monitorsubscriber.py` BEFORE renaming you have to revert the migration and then rename and apply. Otherwise your DB will become 💩

Reply:
Renamed to `0002_add_fxa_uid_to_invitation.py`